src/pickup.cpp

```diff
@@ -784,18 +800,39 @@ void Pickup::pick_up( const tripoint &p, int min )
                                  c > 0; ++it, --c ) {
                                 charges += it->_item.charges;
                             }
-                            item_name = string_format( _( "%s of %s" ),
-                                                       stacked_here[true_it].begin()->_item.display_money( getitem[true_it].count, charges ),
-                                                       format_money( charges_total ) );
+                            if( stealing ) {
+                                item_name = string_format( _( "%s %s of %s" ), stolen,
```

Comment:
Translator comments would be welcome.

Reply:
The stealing string addition is just a red "!" added to the beginning of the item string. I havnt changed anything that would require translation...

Reply:
Unless you mean,,, the actual item description string, pre-existing code? , I can add pggettext to that, if thats what you mean?

Reply:
I want an example of what this `%s %s of %s` expands into.

Reply:
Yeah, It's cash cards, and the similar patten follows in the other strings below, so I've added a comment to that effect.